views/build.gradle

```diff
@@ -23,13 +23,31 @@ android {
 
 dependencies {
 	compile fileTree(dir: 'libs', include: ['*.jar'])
+
+	// Let's ensure that we are using only one support library version
+	compile "com.android.support:design:$supportlib_version"
+	compile "com.android.support:support-v4:$supportlib_version"
 	compile "com.android.support:appcompat-v7:$supportlib_version"
-	compile 'uk.co.chrisjenx:calligraphy:2.3.0'
-	compile 'com.github.matt-allen:Android-Universal-Image-Loader:2.1.2'
-	compile 'com.yqritc:android-scalablevideoview:1.0.4'
 	compile "com.android.support:recyclerview-v7:$supportlib_version"
+	compile "com.android.support:transition:$supportlib_version"
+	compile "com.android.support:support-annotations:$supportlib_version"
+	compile "com.android.support:support-v4:$supportlib_version"
```

Comment:
Duplicate definition, the same lib defined in line 29

Reply:
Fixed in 59c8df31ce34488936f06686be90d85060c2971e